docs/aboutme.md

```diff
@@ -30,15 +25,22 @@ subtitle:
 * [Monique Moledo](https://github.com/MoniMoledo)
 * [Victor Phung](https://github.com/starmon00)
 * [Xi Zhang](https://github.com/xizzzz)
+* [Hao Chen](https://github.com/haochen07)
+* [Vignesh Sankar](https://github.com/vignesh-sankar)
+* [Shengjie Xu](https://github.com/HotLemonJuice)
+* [Liangju Chu](https://github.com/liangjuc)
+* [Sicong Liu](https://github.com/lsclovecode)
+* [Vidhyasagar Thirumaraiselvan](https://github.com/vidhya567)
+* [Haohan Zhang](https://github.com/hh1680651)
 
 
 ## Acknowledgement
-This work has been supported by NIH award 1U01HG008488-01, by NSF CNS award 1305430, 
-and by the Army Research Laboratory under Cooperative Agreement Number W911NF-16-2-0110. 
+This work has been supported by NIH award 1U01HG008488-01, by NSF CNS award 1305430,
```

Comment:
Keep the first "by" and remove the other "by"s.

Reply:
done